.circleci/config.yml

```diff
@@ -1,6 +1,8 @@
 ---
 # currently just a thin wrapper around real_config to add dynamic parameters
-version: '2.1'
+# 
+# For more information, see this directory's README.md
+version: 2.1
```

Comment:
Change made because:
* my IDE prefers it unquoted
* CircleCI docs don't quote the version

Reply:
That makes a difference in some situations, since YAML parsers vary in what they do with auto type detection. This is a float now, but was a string before. I've been burned in, for example, helm Charts when bumping a minimum version comparison value from 1.2.3 to 1.3 - suddenly the string is a float and some random junk breaks. 😂